docs/cloud/1-services/2-lightdb/3-read-write-data.md

```diff
@@ -17,6 +17,21 @@ Devices interact with LightDB using the gateways available on the platform. For
 >
 > /.d/temp/active
 
+Another way of interaction with the platform is opening a Real-Time WebSocket connection only to listen to any changes in a device state path:
+
+| Method    | Description                       | Complete Endpoint                                               |
+| --------- | --------------------------------- | --------------------------------------------------------------- |
+| WebSocket | Listening to LightDB Device State | wss://api.golioth.net/v1/ws/projects/{projectId}/devices/{deviceId}/data{/path=\*\*}?x-api-key={API_KEY} |
```

Comment:
Users can also use the `jwt` query param and use the JWT based API Keys from our platform.

Reply:
Could you show me how the JWT is embedded in the connection request? Does it use the same x-api-key parameter or another way?